graphql-protocol/src/main/java/com/linecorp/armeria/server/graphql/protocol/AbstractGraphqlService.java

```diff
@@ -190,22 +255,23 @@ private static String toStringFromJson(String name, @Nullable Object value) {
     @SuppressWarnings("unchecked")
     private static Map<String, Object> toMapFromJson(@Nullable Object maybeMap) {
         if (maybeMap == null) {
-            return ImmutableMap.of();
+            return new HashMap<>();
```

Comment:
This is inefficient because `extensions` might be null with high probability.

Reply:
Because of the `VariableMapper` we need a mutable collection. So, do we need to take an additional argument and set a default value? 🤔